### PR TITLE
Release v1.1.0

### PR DESCRIPTION
**Pull request by lucor**
This fyne-cross release updates to Go v1.16.4, adds support for macOS and FreeBSD arm64 builds along with other improvements and fixes.

### Added

- Add darwin arm64 target #39
- Add FreeBSD on arm64 target #29
- Add the `darwin-image` command to build the darwin docker image
- Add the `local` flag for darwin to build directly from the host
- Add a dedicated docker image for macOS
- Add a dedicated docker image for Windows

### Changed

- Update Go to v1.16.4
- Update fyne CLI to v2.0.3
- Update FreeBSD SDK to v12.2 #29
- Refactor docker images layout to ensure compatibility with previous versions of fyne-cross

### Fixed

- Fix android keystore path is not resolved correctly
- Fix some release flags are always set even if empty
- Fix appID flag should not have a default #25
- Fix the option --env does not allow values containing comma #35

### Removed

- Remove darwin 386 target
- Remove the dependency from the docker/golang-cross image for the base image

### CODE REVIEW

**Review by Jacalz — COMMENTED** [review]
I think we might want to get Go 1.16.4 into this before release?

**Comment by lucor** [comment]
Updated Go to 1.16.4 fyne CLI to 2.0.3 in 70c0c69

**Comment by Jacalz** [comment]
Don't wait for my review on this before merging. I won't be able to look at it for a little while.

**Review by andydotxyz — APPROVED** [review]

**Pull request merged**
